Cli 1.6 docs follow-up (#1289)
astronomer/docs: astro/develop-project.md

    variable_value: <my-variable-value-2>
```

> Save the modified `airflow_settings.yaml` file, start your local environment, and then run: 

Reviewer: ```suggestion
Save the modified `airflow_settings.yaml` file and run:
```

Author: We need to have the step for starting a local environment because this command doesn't work without that

> Once you save these values in your `airflow_settings.yaml`, start your local environment and run: 

Reviewer 2: ```suggestion
Save the modified `airflow_settings.yaml` file, start your local environment, and then run: 
```
